dashboard/app/models/ability.rb

```diff
@@ -132,6 +133,7 @@ def initialize(user)
         can :manage, :pd_workshop_summary_report
         can :manage, :pd_teacher_attendance_report
         can :manage, Pd::TeacherApplication
+        can :manage, :pd_workshop_user_admin
```

Comment:
My understanding is that this controller will implement features that allow an internal user to administer the relationship between Workshop users and various Workshop features (assign Courses to Facilitators, assign Regional Partner to Organizer, etc.).  Most of these features are not directly related to permission, so I renamed it to `workshop_user_admin` to indicate that the functionality is  "Workshop - User Administration".  Would a comment in the controller help, or is there an alternate name that fits?